fe/fe-core/src/main/java/org/apache/doris/nereids/glue/translator/PhysicalPlanTranslator.java

```diff
@@ -415,12 +416,20 @@ public PlanFragment visitPhysicalOlapTableSink(PhysicalOlapTableSink<? extends P
             slotDesc.setIsNullable(column.isAllowNull());
             slotDesc.setAutoInc(column.isAutoInc());
         }
-        OlapTableSink sink = new OlapTableSink(
+        OlapTableSink sink;
+        // This statement is only used in the group_commit mode in the http_stream
+        if (context.getConnectContext().isGroupCommitStreamLoadSql()) {
```

Comment:
Here, can we set the session variable `group_commit` in `FrontendServiceImpl` and use here？
In this way, we does not need add a new variable in ConnectContext